server/src/main/java/io/druid/segment/loading/SegmentLoaderLocalCacheManager.java

```diff
@@ -104,53 +104,71 @@ public File getSegmentFiles(DataSegment segment) throws SegmentLoadingException
   {
     StorageLocation loc = findStorageLocationIfLoaded(segment);
 
-    final File retVal;
+    File retVal = null;
 
     if (loc == null) {
```

Comment:
Maybe move this `if  {}` body into a separate method, to make `getSegmentFiles()` more readable?